scripts/tools/common.yml

```diff
@@ -109,8 +109,15 @@ etors:
       desc: $t_tracer_exp_desc_t
       value: "0x00010001"
     - name: METRICS_CALCULATE_EXP_DESC
-      desc: $t_metric_calculate_exp_desc_t
+      desc:
```

Comment:
we have other places with also mixed names, like above:

GLOBAL_METRICS_TIMESTAMPS_EXP_PROPERTIES

would we want to change it on this PR or another one?

Reply:
updated